zebrad/tests/common/lightwalletd/wallet_grpc.rs

```diff
@@ -42,7 +42,7 @@ pub fn spawn_lightwalletd_with_rpc_server(
 
     lightwalletd.expect_stdout_line_matches("Starting gRPC server")?;
     if wait_for_blocks {
```

Comment:
This condition might not be needed any more, but we can check that when we get to the other grpc tests.